### PR TITLE
Tx 5

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+release: python manage.py migrate --no-input
```
Comment (iamr0b0tx) [inline Procfile:1]: no-input?

**Review by iamr0b0tx — APPROVED** [review]

**Comment by osadolortech** [comment]
so Heroku is automatically migrating, once I push my repository

File: Procfile (hunk 1 above)
Comment (iamr0b0tx) [reply Procfile:1]: where did you learn `no-input`?

**Comment by osadolortech** [comment]
I learnt it from a video online, I could not find much detailed information
on the DJango documentations, I now wonder if --no-input is a 'good' idea?

**Comment by iamr0b0tx** [comment]
Ok. merge when you are done

**Pull request merged**